packages/terra-spacer/src/Spacer.module.scss

```diff
@@ -14,319 +14,319 @@ $large-plus-4: 3.5714285714rem;
   }
```

Comment:
I would get feedback from @neilpfeiffer on the changes in this file before we officially merge into master. This file might be a rare exception where we don't want to theme the properties individually. 

`--terra-spacer-small-minus-2`  is probably expected be consistent with margin-top, bottom, left, right, etc.. The current implementation allows all the properties for small-minus-2 to be different. 

We can style lint ignore certain rules in this file if it's decided we want to preserve the previous theme strategy.

Reply:
revert: https://github.com/cerner/terra-core/pull/2125/commits/ac1228ea08845793b154c2f5e5c6377e29804096
disable: https://github.com/cerner/terra-core/pull/2125/commits/5c574263d2b9623f566e5a204c0aee960e26aadb